holes.toml

```diff
@@ -158,6 +158,40 @@ preamble = '''
     and execute each program in turn.
 '''
 
+['Boxes Intersection']
+experiment = 1
```

Comment:
this is the github issue number of where to leave feedback, its a bit of a chicken and egg problem but i suggest creating the issue first, nothing the issue number in the PR so it gets referenced on the issue page, and updating this number to be that issue number.

Reply:
ah I see!
yes, no problem, here it is #281